CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ----------------
 
 - Changed output of the current temperature
+- Added parameter `--statistics` to display the hottest and coldest temperature each day of the last 7 days
```

Comment:
Please add new entries above the latest item, because the file is read from top to bottom (sorted by date in descending order).